src/Solutions/Folds.idr

```diff
@@ -90,6 +86,11 @@ joinTR = go Lin
         go sx []        = sx <>> Nil
         go sx (x :: xs) = go (sx <>< x) xs
 
+-- Or using the connection between join and bind:
+-- joinTR xss = bindTR xss id
```

Comment:
Please uncomment your version and give it a new name to get typechecking and syntax highlighting.

Reply:
done: 2b77f31d55888521fb0995730ddd7482715f4e36